Source/NSURL.m

```diff
@@ -1575,19 +1575,20 @@ - (NSString*) host
     {
       char	buf[strlen(myData->host)+1];
 
-      if (*myData->host == '[')
-	{
-	  char	*end = unescape(myData->host + 1, buf);
-
-	  if (end[-1] == ']')
-	    {
-	      end[-1] = '\0';
-	    }
-	}
-      else
-	{
-          unescape(myData->host, buf);
-	}
+  //     if (*myData->host == '[')
```

Comment:
If this code is not needed anymore it should be removed not commented out.

Reply:
Good catch! I will clean that up.